examples/django-deployment/Makefile

```diff
@@ -22,10 +22,39 @@ test-and-coverage:
 .PHONY: compile-packages
 compile-packages:
 	PIPENV_VERBOSITY=-1                                \
-	mkdir -p ./artifacts && 									 \
+	mkdir -p ./artifacts && 									 				 \
 	cd ../../connectors/django/	&&  	 		        	   \
 	pipenv run python setup.py sdist bdist_wheel &&    \
 	cp -r dist/* ../../examples/django-deployment/artifacts/
-	cd ../../fma-core	&&  	          							 \
+	cd ../../fma-core	&&  	          							 	 \
 	pipenv run python setup.py sdist bdist_wheel &&    \
 	cp -r dist/* ../examples/django-deployment/artifacts/
+
+
+.PHONY: complete-install
+complete-install:
+	make compile-packages && 																										 \
+	make install && 																														 \
+	unameOut="$(uname -s)" && 																									 \
+		case "${unameOut}" in  																										 \
+			Linux*)     apt-get install redis;; 																		 \
+			Darwin*)    brew install redis;; 																				 \
+      CYGWIN*)    echo "Error! redis not supported by ${unameOut}"; exit 125;; \
+      MINGW*)     echo "Error! redis not supported by ${unameOut}"; exit 125;; \
+      *)          echo "Error! redis not supported by ${unameOut}"; exit 125;; \
+  esac
+
+
+.PHONY: start-service-local
+start-service-local:
+	export FMA_SETTINGS_MODULE=federated_learning_project.fma_settings && 			 \
+	pipenv run python manage.py migrate && 																		 \
+	pipenv run python manage.py createsuperuser
```

Comment:
I would consider these part of install and not start as we only need to do these once.